references/sdk_models.md

```diff
@@ -23,7 +23,6 @@ Data sample
 - data_manager_keys: Optional[List[str]]
 - path: Optional[DirectoryPath]
 - creation_date: datetime
-- test_only: bool
```

Comment:
Is this left over from something else?

Reply:
Wooops this is a rebase issue I think

Reply:
Ah no this is due to the make doc command, that was not done in the previous PR... On substra, you can skip the .md files in the review as they are automatically generated